stackslib/src/chainstate/stacks/db/mod.rs

```diff
@@ -1132,6 +1133,15 @@ impl StacksChainState {
                         tx.execute_batch(cmd)?;
                     }
                 }
+                "7" => {
+                    // migrate to nakamoto 3
```

Comment:
Nit

```suggestion
                    // migrate to nakamoto 5
```

Reply:
While you're at it, may as well change the one above to 4 as well.

Reply:
I just removed the comments -- at best, they're just restating the log lines beneath them.